src/controller/CHIPDeviceController.cpp

```diff
@@ -706,8 +706,7 @@ CHIP_ERROR DeviceCommissioner::EstablishPASEConnection(NodeId remoteDeviceId, Re
         }
     }
 #endif
-    // TODO: In some cases like PASE over IP, SAI and SII values from commissionable node service should be used
-    session = mSystemState->SessionMgr()->CreateUnauthenticatedSession(params.GetPeerAddress(), device->GetRemoteMRPConfig());
+    session = mSystemState->SessionMgr()->CreateUnauthenticatedSession(params.GetPeerAddress(), params.GetMRPConfig());
```

Comment:
I wonder... Should we call `device->UpdateDeviceData` somewhere here too?  Or maybe `CommissioneeDeviceProxy::Init` should just take the MRPConfig as an argument?

Reply:
I have ended up calling `device->UpdateDeviceData` but I'm not sure who is using the map configfrom the `PASESession` at the moment.